addons/osfstorage/views.py

```diff
@@ -96,6 +96,27 @@ def osfstorage_update_metadata(payload, **kwargs):
 
     return {'status': 'success'}
 
+@must_be_signed
+@decorators.load_guid_as_target
+def osfstorage_get_storage_quota_status(target, **kwargs):
+    # Storage caps only restrict Nodes
+    if not isinstance(target, Node):
+        return {
+            'over_quota': False
+        }
+    # Storage calculation for the target has been accepted and will run asynchronously
+    if target.storage_usage is None:
+        raise HTTPError(http_status.HTTP_202_ACCEPTED)
+
+    # Storage cap limits differ for public and private nodes
+    if target.is_public:
+        over_quota = False if target.storage_limit_status < 4 else True
+    else:
+        over_quota = False if target.storage_limit_status < 2 else True
+    return {
```

Comment:
A request and a golf:

**The request**: can we name these magic numbers?  Are there constants somewhere we can import?

**The golf**: `over_quota = not(target.storage_limit_status < 4)` or `over_quota = target.storage_limit_status >= 4`. Not a dealbreaker, though.

Reply:
Updated!